frontend/src/app/homepage-offer/offers.service.ts

```diff
@@ -41,4 +48,24 @@ export class OffersService {
     return `${environment.djangoRoot}/offers/${offer.slug}/${offer.id}/join`;
   }
 
+  postOffer(offer: Offer) {
+    console.log(offer)
+    return this.http.post(`${environment.apiRoot}/offers/`, offer, { withCredentials: true })
+    .map(response => {
+      if (response.status === 201) {
```

Comment:
I know it is not a first time in our codebase where magic numbers for response status appear, but I think we should create the enum for response status codes.